extensions/amp-ad-network-adsense-impl/0.1/test/test-amp-ad-network-adsense-impl.js

```diff
@@ -90,6 +92,18 @@ describes.realWin(
       impl = new AmpAdNetworkAdsenseImpl(element);
       impl.win['goog_identity_prom'] = Promise.resolve({});
       isResponsiveStub = sandbox.stub(impl, 'isResponsive_');
+      // Make tests faster by completing timeouts instantly.
+      sandbox.stub(Services, 'timerFor').returns({
+        timeoutPromise: (unused, promise) => {
+          if (promise) {
+            return promise;
+          }
+          return Promise.reject(new Error('No token'));
+        },
+      });
+      return Services.storageForDoc(element).then(storage => {
+        storageMock = sandbox.mock(storage);
```

Comment:
for test doubles, we now prefer `stub + spy` over `mocks`

Reply:
done